MatrixSDK/Space/MXRoomAliasAvailabilityChecker.swift

```diff
@@ -84,4 +84,17 @@ public extension String {
     func toValidAliasLocalPart() -> String {
         return lowercased().replacingOccurrences(of: " ", with: "-").filter { MXRoomAliasAvailabilityChecker.validAliasCharacters.contains($0) }
     }
+    
+    /// Extract the valid local alias part String of the string ((e.g. "my_alias" for the string "#my_alias:example.org")
+    /// - Returns:the valid local alias part extracted from the string.
+    func extractLocalAliasPart() -> String {
```

Comment:
I'm not sure I like these methods being available on any kind of string. Is there a way to limit them to aliases? Maybe move them to MXTools and have them receive the string from the outside?